src/test/scala/centaur/StandardTestCaseSpec.scala

```diff
@@ -28,8 +28,24 @@ class StandardTestCaseSpec extends FlatSpec with Matchers with ParallelTestExecu
   }
 
   def executeStandardTest(testCase: StandardTestCase, f: Workflow => Test[_]): Unit = {
-    it should s"${testCase.testFormat.testSpecString} ${testCase.workflow.name}" in {
-      f(testCase.workflow).run.get
+
+    // Make tags. Enforce lowercase:
+    // TODO: After #45 merges, can replace the testFormat.toString.stripSuffix("Test") with just .name -
```

Comment:
Can we count on you to make this a TODONE when #45 merges, seeing how you're the author of both? :)